core/indexer/common.go

```diff
@@ -187,6 +187,17 @@ func (cm *commonProcessor) buildRewardTransaction(
 }
 
 func (cm *commonProcessor) convertScResultInDatabaseScr(sc *smartContractResult.SmartContractResult) ScResult {
+	encodedData := strings.Split(string(sc.Data), "@")
+
+	decodedData := ""
+	for _, enc := range encodedData {
+		val, err := hex.DecodeString(enc)
+		if err != nil || len(val) == 0 {
+			continue
```

Comment:
add enc to the list if it is not possible to decode

Reply:
done